SuperblocksHQ/superblocks-lab
# Connect project list

### Description of the Change

- Connected all the project update details and remove epics to the UI.
- Minor design updates to harmonize colors. 

### Verification Process

- Open the dashboard
- Go to a project
- Enter the project settings
- Update the project name
  - The name on the breadcrumbs should update to the new one
  - A success toast should appear
